start

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+python /Project-env/Project/bin/manage.py redcap json /Input/metadata.csv 
```

Comment:
Update the paths here. The working directory is already set to `/usr/src/app/project` in the Dockerfile, so these paths can be simplified to be relative to that, e.g. `python bin/manage.py ...`